uom_dp/models/uom_uom.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class UoM(models.Model):
+    _inherit = "uom.uom"
+
+    uom_dp = fields.Char("Demical Places", compute="_compute_uom_dp")
+
+    @api.multi
+    def _compute_uom_dp(self):
+        for uom in self:
+            uom.uom_dp = str(uom.rounding)[::-1].find(".")
```

Comment:
Now that you have prepared a new field, should we make the logic (and the field name) consistent with this? https://github.com/odoo/odoo/blob/904a46f28cb64e3bd51c40dc364b256d434d1c93/odoo/addons/base/models/res_currency.py#L71-L78